single-host-httpclient/src/main/java/com/github/nhenneaux/resilienthttpclient/singlehostclient/ServerConfiguration.java

```diff
@@ -10,6 +12,8 @@ public class ServerConfiguration {
     private static final long DEFAULT_CONNECTION_HEALTH_CHECK_PERIOD_IN_SECONDS = 30;
     private static final long DEFAULT_HEALTH_READ_TIMEOUT_IN_MILLISECONDS = TimeUnit.SECONDS.toMillis(5);
     private static final int DEFAULT_FAILURE_RESPONSE_COUNT_THRESHOLD = -1; // It means no validation by failed response count
+    public static final Consumer<HttpRequest.Builder> DEFAULT_REQUEST_TRANSFORMER = request -> {
```

Comment:
Doesn't need to be `public`, does it?

Reply:
As discussed in other comments - this variable is accessed from multiple places so made public.